x12-edi/2.6/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,78 @@
+= X12 EDI Connector 2.6 - Mule 4
+:page-aliases: connectors::x12-edi/x12-edi-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium] +
+
+Anypoint Connector for X12 EDI (X12 EDI Connector) enables you to convert X12 messages to and from DataWeave-compatible representations by using lists and maps.
+All of the schema definitions you need for xref:x12-edi-versions-x12.adoc[supported X12 standard transaction sets] are included. For complete information about schemas, see xref:x12-edi-connector::x12-edi-schema-language-reference.adoc[X12 EDI Schema Language Reference].
+
+Many Health Insurance Portability and Accountability Act (HIPAA) document versions are also supported, as listed separately in the xref:x12-edi-versions-hipaa.adoc[Supported HIPAA Versions]. For more information about HIPAA usage, refer to xref:x12-edi-connector-hipaa.adoc[Connector HIPAA Standards].
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+To use X12 EDI connector, you must have:
+
+* Purchased a MuleSoft license for Anypoint B2B
+* Anypoint Studio 7.0 or later
+
+[NOTE]
+
+X12 EDI Connector supports files of up to 15 MB in size, and the memory requirement is approximately 40 to 1.
+For example, a 1 MB file requires up to 40 MB of memory to process, so it's important to consider this memory requirement in conjunction with your Transactions Per Second (TPS) needs for large X12 files.
+This is not an exact figure; the value varies according to the complexity of the mapping instructions.
+
+== Exchange Templates and Examples
+
+https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates
+that you can use as starting points for your apps and examples that illustrate a complete solution.
+
+=== X12 EDI Connector Templates in Exchange
+
+https://www.mulesoft.com/exchange/org.mule.templates//[Anypoint Exchange Templates] provide starting points for Studio projects such as:
+
+* Sets up your Outbound transactions.
+
+https://www.mulesoft.com/exchange/org.mule.templates/template-b2b-edi-outbound-x12-map/[Map application messages to outbound EDI X12 transactions]
+
+* Sets up your Inbound transactions.
+
+https://www.mulesoft.com/exchange/org.mule.templates/template-b2b-edi-inbound-x12-map/[Map inbound EDI X12 transactions to application message format]
+
+* Use this for setting up your Partner Manager content storage.
```

Comment:
The lead-in sentence to this bullet point is:
[Anypoint Exchange Templates] provide starting points for Studio projects such as:

So this wording should probably be modified somehow.